src/osm_configurator/view/toplevelframes/data_frame.py

```diff
@@ -294,7 +295,7 @@ def __select_cut_out(self):
         """
         Opens the explorer letting the user choose a file selecting the cut-out
         """
-        chosen_path: Path = self.__open_explorer(list["png"])  # TODO: insert accepted filetypes
+        chosen_path: Path = self.__get_directory_path()
```

Comment:
Why do you ask the user for a directory when selecting a cut out file? It was correct, the way it was before